common/model/pages.js

```diff
@@ -1,9 +1,11 @@
 // @flow
 import type { GenericContentFields } from './generic-content-fields';
+import type { Link } from './link';
 
 export type Page = {|
   type: 'pages',
   ...GenericContentFields,
+  onThisPage: Link[],
```

Comment:
didn't you add this to the GenericContentFields, which it probably isn't so maybe remove from there?

Reply:
I did add it to the `GenericContentFields` and I didn't feel great about it. The reason I did is that they're derived from the parsing of the body, which itself is a generic field. I'll see about moving it out, though.